frontend/package.json

```diff
@@ -6,8 +6,12 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.6.8",
+    "pexels": "^1.4.0",
     "react": "^18.2.0",
+    "react-axios": "^2.0.6",
     "react-dom": "^18.2.0",
+    "react-redux": "^9.1.0",
```

Comment:
We are not going to use react redux in this project it's too complex for a project this simple

Reply:
I removed react-redux.